parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -407,8 +423,8 @@ ensureAddons() {
 {{- end}}
 {{- if not HasCustomPodSecurityPolicy}}
   retrycmd 120 5 30 $KUBECTL get podsecuritypolicy privileged restricted || exit_cse {{GetCSEErrorCode "ERR_ADDONS_START_FAIL"}} $GET_KUBELET_LOGS
-  rm -Rf ${ADDONS_DIR}/init
 {{- end}}
+  rm -Rf ${ADDONS_DIR}/init
```

Comment:
This rm statement should have always been outside the PSP if block, we want to do it every time after we process all addons from init directory (the above three if blocks do so).